new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiUpdateDomainConfig.java

```diff
@@ -285,20 +335,208 @@ public void beforeEach() {
     }
   }
 
+  /**
+   * Create a WebLogic domain with a defined configmap in configuration/model 
+   *  section of the domain resource.
+   * The configmap has multiple sparse WDT model files that define 
+   *  a JDBCSystemResource, a JMSSystemResource and a WLDFSystemResource.
+   * Verify all the SystemResource configurations using the rest API call 
+   *   using adminserver's public nodeport
+   */
+  @Test
+  @Order(1)
+  @DisplayName("Verify the pre-configured SystemResources in a model-in-image domain")
+  public void testMiiCheckSystemResources() {
+    ExecResult result = null;
+    result = checkSystemResourceConfiguration("JDBCSystemResources", "TestDataSource");
+    assertNotNull(result, "CheckJDBCSystemResources returned null");
+    logger.info("CheckJDBCSystemResource returned {0}", result.toString());
+    assertEquals("200", result.stdout(), "JDBCSystemResource not found");
+    logger.info("Found the JDBCSystemResource configuration");
+
+    result = null;
+    result = checkSystemResourceConfiguration("JMSSystemResources", "TestClusterJmsModule");
+    assertNotNull(result, "CheckJMSSystemResources returned null");
+    logger.info("CheckJMSSystemResource returned {0}", result.toString());
+    assertEquals("200", result.stdout(), "JMSSystemResource not found");
+    logger.info("Found the JMSSystemResource configuration");
+
+    result = null;
+    result = checkSystemResourceConfiguration("WLDFSystemResources", "TestWldfModule");
+    assertNotNull(result, "CheckWLDFSystemResources returned null");
+    logger.info("CheckWLDFSystemResource returned {0}", result.toString());
+    assertEquals("200", result.stdout(), "WLDFSystemResource not found");
+    logger.info("Found the WLDFSystemResource configuration");
+
+    result = null;
+    result = checkJdbcRuntime("TestDataSource");
+    logger.info("checkJdbcRuntime: returned {0}", result.toString());
+    assertTrue(result.stdout().contains("jdbc:oracle:thin:localhost"),
+         String.format("DB URL does not match with RuntimeMBean Info"));
+    assertTrue(result.stdout().contains("scott"),
+         String.format("DB user name does not match with RuntimeMBean Info"));
+    logger.info("Found the JDBCSystemResource configuration");
+
+  }
+
+  /**
+   * Start a WebLogic domain using model-in-image with JMS/JDBC SystemResource.
```

Comment:
Are we deleting multiple system resources? If so, modify the javadoc to use plural.

Reply:
done